src/services/swapServices/index.ts

```diff
@@ -86,7 +92,11 @@ export const swapNativeForAssetExactIn = async (
           if (response.dispatchError.isModule) {
             const decoded = api.registry.findMetaError(response.dispatchError.asModule);
             const { docs } = decoded;
-            dotAcpToast.error(`${docs.join(" ")}`);
+            if (checkIfExactError(docs.join(" "))) {
+              dotAcpToast.error(t("swapPage.slippageError"));
+            } else {
+              dotAcpToast.error(`${docs.join(" ")}`);
+            }
```

Comment:
Also here. Instead of the whole if/else statement u can put

 dotAcpToast.error(checkIfExactError(docs.join(" ") ? t("swapPage.slippageError") : `${docs.join(" ")}`);